tree-sitter.json

```diff
@@ -0,0 +1,43 @@
+{
+  "grammars": [
+    {
+      "name": "fortran",
+      "camelcase": "Fortran",
+      "scope": "source.fortran",
+      "path": ".",
+      "file-types": [
+        "f90",
+        "F90",
+        "f95",
+        "F95"
+      ],
+      "highlights": [
+        "queries/highlights.scm"
+      ],
+      "injection-regex": "fortran"
+    }
+  ],
+  "metadata": {
+    "version": "0.2.0",
+    "license": "MIT",
+    "description": "Fortran grammar for tree-sitter",
+    "authors": [
```

Comment:
If you feel inclined to you can add yourself to the authors listing, you have really advanced the state of this library.

Reply:
Thanks! I've done so, looks like the commit is taking some time to turn up though